app-backend/api/src/main/scala/maptoken/QueryParams.scala

```diff
@@ -0,0 +1,19 @@
+package com.azavea.rf.api.maptoken
+
+import java.util.UUID
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.directives.ParameterDirectives.parameters
+import com.azavea.rf.database.query.MapTokenQueryParameters
+
+import com.azavea.rf.api.utils.queryparams._
+
+/** Trait to mix in for image specific query parameters */
+trait MapTokensQueryParameterDirective extends QueryParametersCommon {
+
+  val mapTokenQueryParams = parameters(
+    'name.as[String].?,
```

Comment:
We should have the ability to filter by `created_by` and `organization`, so that management pages can list all relevant tokens using a single paginated request.